src/vue/modules/issuances-explorer/store/index.js

```diff
@@ -0,0 +1,48 @@
+import { Issuance } from '../wrappers/issuance'
+
+import { types } from './types'
+
+import { Sdk } from '@/sdk'
+import { OP_TYPES } from '@tokend/js-sdk'
+
+export const state = {
+  accountId: '',
+  issuances: [],
+}
+
+export const mutations = {
+  [types.SET_ACCOUNT_ID] (state, accountId) {
+    state.accountId = accountId
+  },
+  [types.SET_ISSUANCES] (state, issuances) {
+    state.issuances = issuances
+  },
+  [types.CONCAT_ISSUANCES] (state, issuances) {
+    state.issuances = state.issuances.concat(issuances)
+  },
+}
+
+export const actions = {
+  [types.LOAD_ISSUANCES] ({ getters }) {
+    // TODO: Use API caller when the new endpoint is able
```

Comment:
The new endpoint will never be able to filter operations by their type. You need to fetch requests now.

Also new modules must not use the old API version at all. If you don't have the endpoint you need, pls inform the backend team about this, because everyone thinks the API is ready

Reply:
Updated, used new endpoint